sandbox/modules/identity/src/test/java/org/opensearch/identity/remotecluster/UserIT.java

```diff
@@ -112,4 +114,106 @@ public void testInternalUsersApi() throws Exception {
 
     }
 
+    public void testResetPasswordApi() throws Exception {
+
+        String username = "test-user";
+        String userCreationContent = "{ \"password\" : \"test\","
+            + " \"attributes\": { \"attribute1\": \"value1\"},"
```

Comment:
Are attributes and permissions required for this call?

Reply:
No, we dont need that and I will remove it with the new commit of refactoring the test